libs/javalib/api/src/mill/javalib/api/JvmWorkerUtil.scala

```diff
@@ -111,6 +111,15 @@ object JvmWorkerUtil {
       case _ => false
     }
 
+  /**
+   * @return true if scala-library contains both Scala 2 and 3 libraries, as opposed to
+   * having both scala3-library and scala2-library.
+   */
+  def isUnifiedLibraryAvailable(scalaVersion: String): Boolean =
+    scalaVersion match {
+      case Scala3Version(minor, _) => minor.toInt >= 8 // 3.8 or later
+    }
```

Comment:
Looks like this match is incomplete, e.g. it will fail for all non-`Scala3Version`s.